src/components/Todo/Todo.js

```diff
@@ -1,8 +1,65 @@
-import React from 'react'
+import React, { useState } from 'react'
 import ListItem from 'components/ListItem/ListItem'
+import Input from 'components/Input/Input'
+import Button from 'components/Button/Button'
+import { FaEdit } from 'react-icons/fa'
+import { MdCancel } from 'react-icons/md'
 
-const Todo = ({ children, ...props }) => {
-  return <ListItem {...props}>{children}</ListItem>
+const Todo = ({ onUpdateText, id, value, children, ...props }) => {
+  const [editing, setEditing] = useState(false)
+  const [toDoText, setToDoText] = useState(value)
+  const [changeElement, setChangeElement] = useState(false)
```

Comment:
`changeElement` es un estado binario. Como buena práctica, a los estados binarios se les ponen nombres que sean una pregunta que pueda responderse con sí o no (`isSelected`, `hasItems`, `canDelete`, etc). Por otro lado, en este caso sí está marcando cuando se está editando el todo, por lo que se puede cambiarlo para aclararlo

```suggestion
  const [isEditing, setIsEditing] = useState(false)
```